src/pages/settings/Wallet/PaymentMethodList.js

```diff
@@ -318,25 +333,27 @@ function PaymentMethodList({
 
     return (
         <>
-            <FlatList
-                data={filteredPaymentMethods}
-                renderItem={renderItem}
-                keyExtractor={(item) => item.key}
-                ListEmptyComponent={shouldShowEmptyListMessage ? renderListEmptyComponent : null}
-                ListHeaderComponent={listHeaderComponent}
-                ListFooterComponent={shouldShowAddBankAccount ? renderListFooterComponent : null}
-                onContentSizeChange={onListContentSizeChange}
-                scrollEnabled={shouldEnableScroll}
-                style={style}
-            />
+            <View style={[style, {minHeight: variables.paymentMethodHeight}]}>
```

Comment:
I believe this may have caused https://github.com/Expensify/App/issues/33075 as `variables.paymentMethodHeight` is only equal to 1 item's height, but we can have multiple cards. More details [here](https://github.com/Expensify/App/issues/33075).